rest/model/task.go

```diff
@@ -247,6 +248,20 @@ func (at *APITask) BuildFromService(t interface{}) error {
 				PRClosed:   v.AbortInfo.PRClosed,
 			},
 		}
+
+		annotation, err := annotations.FindOneByTaskIdAndExecution(id, v.Execution)
```

Comment:
Can we do this elsewhere? 
This  db query will be called every time we call buildFromService. Even in places where it is not needed. 
 Since we only need this in the graphql resolver we should do this work there.

Reply:
yes, I can move it to the status resolver